### ISSUE TITLE
rod of the sun king

### ISSUE THREAD

**Issue by teddys** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3956]
http://www.wowhead.com/item=29996/rod-of-the-sun-king#comments
expected situation : anywhere from 2 to 6 ppm
4:42 fight = less than 30 procs with 6 ppm

current situation : http://imgur.com/a/oGtMe 
4:42 fight duration = 89 procs, that's 18 ppm

if double wf is considered an abuse, using this weapon(untill fixed) should be too

**Comment by repae1f** [comment]
need use proc rate % instead ppm, cuz ppm inexact value ( in comments too many different ppms values (from 2 to 12 ppm))
windfury totem bug also do effect for this item now

ps: sorry bad english

**Comment by Qingle** [comment]
In regards to double WF, do you mean WF weapon not sharing CD on 2 weapons?

**Comment by Surveillant** [comment]
Double WF is totally unrelated and has nothing to do with proc rate. That's an abuse because swing timer mechanics don't work properly. It makes no sense to bring that up here. 

Are we really going to enforce this as bug abusing just because it procs a lot? If so, then you all should be ashamed of using DST, madness, shard of contempt, memento, DMF card blue dragon, etc because they all proc way more than retail too. 

Are we just going to ban all the bis items until they are fixed? Fix them, I agree. But you can't say "you're banned for using it, period" while DST and madness are procing just as much.

**Comment by Surveillant** [comment]
There are a lot of variables here, being a main hand weapon with chance on hit. Obviously the windfury proc rate will influence this weapons proc rate, as will sword specialization procs & the passive haste from sunwell items. No doubt that sunwell gear is the reason we see a larger benefit from the proc now than before the release of SWP. However the rate itself has always been the same and never changed. In retail, the passive haste from sunwell gear also had the benefit of increasing the proc rate of chance on hit procs. 

Obviously it's a powerful weapon, but we should see more data before reducing the proc rate all of the sudden. It hasn't been tested on heavily geared SWP rogues much yet. It wasn't such a big deal before sunwell, I don't think it's such a big deal now, it will be stronger than before but its not godlike. A dual glaive rogue should always beat a rogue using rod of the sun king under the same conditions. And I believe that no matter the other gear, that's the case on Archangel, but we need to see more data before jumping to conclusions.

**Comment by Surveillant** [comment]
Also, there is like no hard data whatsoever on the proc rate on retail. I have searched a ton.

**Comment by teddys** [comment]
yeah it's not a big deal, just random t5 item performing better than double glaives
not a big deal at all

http://www.wow-one.com/forum/topic/61526-rod-of-the-sun-king-proc-rate/
it was already fixed, but I guess someone just fked it up again?

**Comment by repae1f** [comment]
Omg, even if ppm in this fight was 6...rogue will be lost only 35-40k damage and still be on glaves rogue level. I think glaves rogue slack in that fight.
I tried 2 months ago report this item. In all trusted sources peoples writes about 25-27% proc. All tests on AA give that %% proc.

**Comment by Surveillant** [comment]
**"I think glaves rogue slack in that fight."**

This. Aggression could have and should have beat me. Sgtglaive has done 3600+ on mother before. Just because I beat a dual glaive rogue, one time, on a short fight, doesn't necessarily prove anything. Lots of variables.

**"yeah it's not a big deal, just random t5 item performing better than double glaives
not a big deal at all"** 

Who said it outperforms double glaives? Do you have any evidence besides this single mother kill? You have to factor in portals, RNG, buffs, troll vs. blood elf racial, cooldown timing, proc timing, and many many other factors. He's using madness vs. me using warp spring coil. Uptime differences. Tons of variables. It doesn't outperform dual-glaives, everything else constant, simple as that. 

On retail, this item was still 2nd bis, just like it is on Archangel. This item was still extremely powerful and while it may be T5, it was always 2nd bis and that has never changed or been disputed. The proc rate may change but even at 3 ppm it's still going to be 2nd bis. What tier an item comes from is irrelevant, in terms of how powerful it should be. It's a general idea of an item's power, not an exact specification. 

**"http://www.wow-one.com/forum/topic/61526-rod-of-the-sun-king-proc-rate/
it was already fixed, but I guess someone just fked it up again?"**

This is just a random post, this guy doesn't even say if he has used it or tested it himself or give any real information on it. From the people who I know have used it here on Archangel, it's always been exactly like this. And dual-glaives has always been better. I'm pretty sure (in fact 99%) that the proc rate has never been changed, at least in the last 2 years. This random post from multiple years ago doesn't necessarily prove anything either.

**Discussion**

How is this proc rate deal any different from discussions on DST, windfury, madness, memento, DMCard: Blue Dragon, etc etc. - feenix has higher proc rates on everything. We all know this. Yet I still see you using Madness of the Betrayer, despite it being obviously way more OP than retail and having a much higher proc rate. That means if using Rod is bug abusing, surely using any of these items would be considered such. 

Can we just agree on one thing? We need more data. I personally don't agree with changing it before more data is gathered. And if the proc gets reduced, that's fine. What's not fine, is enforcing a ban on using certain items because of the current proc rate. That's a dangerous precedent to set, and opens up every item to the same treatment. Do you want to be forced to keep Madness in your bags until its fixed?

Some of these items may never be fixed. Should we just keep waiting and using lower-quality items?

I will continue to use Rod of the Sun King, an item I paid for with reputation points from the shop, until a staff members informs me that it's against the rules.

**Comment by Surveillant** [comment]
Just look at any Ashes Incarnate raid where Paroxysmx and Kildrazien are both present. Kildrazien always wins on the DPS meters, and their gear is about identical, besides Paro having RotSK vs. Kildrazien's glaives. 

RotSK is good, but it's not legendary good.

**Comment by repae1f** [comment]
> DST, windfury, madness, memento, DMCard: Blue Dragon, etc etc. - feenix has higher proc rates on everything

dst and memento is ok
